scripts/launch_cluster.py

```diff
@@ -684,22 +702,21 @@ def launch(type, id, secret, region, output):
     to get access your clusters' information or create a Kubernetes cluster automatically, finally output
     a kube config of the cluster.
 
-    This script assumes that kubectl and aws-iam-authenticator are in your PATH.
+    This script assumes that kubectl in your PATH and if use AWS cluster, you also need to
+    install aws-iam-authenticator.
     """
 
-    check_file_exist(output)
-    check_kubectl_installed()
+    if not check_requirements(type, output):
+        click.echo(
+            "Requirements of the script not ready, please follow the promp to install requirement."
+       )
+        sys.exit(1)
+    else:
+        click.echo("Requirements fulfilled.")
+
     if type == "aws":
-        if boto3 is None:
-            click.echo(
-                "boto3 module not found, please install by \"pip3 install boto3\"."
-            )
         launcher = AWSLauncher(id, secret, region, output)
     elif type == "aliyun":
-        if CS20151215Client is None:
-            click.echo(
-                "aliyun sdk not found, please install by \"pip3 install alibabacloud_cs20151215 alibabacloud_ecs20140526 alibabacloud_vpc20160428\"."
-            )
         launcher = AliyunLauncher(id, secret, region, output)
     else:
         click.echo("Not support cloud type %s" % type)
```

Comment:
Still proceed to launch cluster if  unsupported cloud type is met.
"Not support" -> "Not supported"?

Reply:
actually， the option can only be aws or aliyun, here will  remove the else branch